### ISSUE TITLE
Template: Buttons should be coded as Links - 4.1.2 Name, Role, Value 

### ISSUE THREAD

**Issue by syarwick** [loyolauniversitychicago/loyola-template-4.0#92]
The buttons at the top of the menu "A-Z
Contact
Directories
Locus
Support LUC" should all be coded as links.

**Comment by syarwick23** [comment]
fixed in new template
